### PR TITLE
Feat: Added History page to my Video library

### PR DESCRIPTION
**Pull request by rahul31agrawal**
Hi Folks , 
I have added History page to my Video library, please give your PR reviews.

Thanks

### CODE REVIEW

File: src/components/sideBar/sideBar.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ const Sidebar = () => {
     </div>
     <div className = "sidebarContents">
       <i className="far fa-history fa-2x"></i>
-      <h3 className = "sideBarText">History</h3>
+      <h3 className = "sideBarText" onClick={() => { navigate("/history")}}>History</h3>
```
Comment (sudosohan) [inline src/components/sideBar/sideBar.js:21]: Use `<Navlink>` component here instead of `useNavigate`

File: src/App.js [hunk 2]
```diff
@@ -4,14 +4,16 @@ import { Routes, Route } from "react-router";
 import {Home} from "./pages/homePage/home";
 import {LikesPage} from "./pages/likesPage/likesPage";
 import {WatchVideo} from "./pages/watchVideo/watchVideo";
+import {HistoryPage} from "./pages/historyPage/historyPage";
 function App() {
   return (
     <div className="App">
       <Routes>
         <Route path="/" element={<Home />} />
         <Route path="/Videos" element={<VideoListing/>} />
```
Comment (sudosohan) [inline src/App.js:13]: Don't use camel casing or capital letters  for paths use small letters

**Review by Tanay-J — APPROVED** [review]
Good work. Checked the preview and Watch History functionality is working as intended.

Suggestion: Please use a code formatter like Prettier.

Approving the changes

File: src/pages/historyPage/historyPage.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        )
+
+         } 
+           
+
+       </div>
+        </>
+    )
+
+
```
Comment (mansipadgelwar) [inline src/pages/historyPage/historyPage.js:42]: remove whitespaces

**Review by mansipadgelwar — APPROVED** [review]
code looks good. you can merge.

**Review by sohan9819 — APPROVED** [review]
Added few suggestions. Rest looks good to merge.

**Pull request merged**